misc_docs/syntax/extension_raw_top_level.mdx

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
One last request: Can you rename this file to `extension_raw_top_level_expression.mdx`? Then we are ready to go!